test-it/ViaTests/DHOArrayStringToNumber.via

```diff
@@ -0,0 +1,101 @@
+// Autogenerated Vireo assembly file
```

Comment:
![Resize](https://user-images.githubusercontent.com/8452083/68043147-a256a680-fca2-11e9-996f-1bde3cc5ee30.png)

Reply:
In the PR to merge this, please include this G test.

Reply:
Until we add support for system configured decimal separator (such as comma instead of period), I wonder whether we should throw an internal Vireo exception for True input for "use system decimal point" boolean input. We could throw a SA error/warning in G if a constant is wired. Might be beyond the scope of the bug you are trying to fix. But, I think we should track this. Thoughts?

We may want to investigate using [toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) in JS and export a function to find the system/locale specific decimal separator and use it here. Format String related nodes may also use the decimal separator.
"Format To String" nodes may also benefit